vmm/src/memory_manager.rs

```diff
@@ -716,6 +768,30 @@ impl MemoryManager {
                 .unwrap();
         }
 
+        // Apply NUMA policy if needed.
+        if let Some(nodemask) = host_numa_nodes {
+            let addr = region.deref().as_ptr();
+            let len = region.deref().size() as u64;
+
+            // Policies are enforced by using MPOL_MF_MOVE flag as it will
+            // force the kernel to move all pages that might have been already
+            // allocated to the proper set of NUMA nodes. MPOL_MF_STRICT is
+            // used to throw an error if MPOL_MF_MOVE didn't succeed.
+            // For now, the implementation supports only 64 NUMA nodes.
```

Comment:
Maybe mention why we have this limitation?

Reply:
I'll take care of that if we still end up with some limitation.